wp-content/themes/inf_theme/admin/class-admin.php

```diff
@@ -18,49 +17,28 @@
  */
 class Admin extends Config {
 
-  /**
-   * General Helper class
-   *
-   * @var object General_Helper
-   *
-   * @since 2.0.1
-   */
-  public $general_helper;
-
-  /**
-   * Initialize class
-   *
-   * @param Helpers\General_Helper $general_helper Helper class instance.
-   *
-   * @since 3.0.0 Removing theme name and version.
-   * @since 2.0.0
-   */
-  public function __construct( General_Helper $general_helper ) {
-    $this->general_helper = $general_helper;
-  }
-
   /**
    * Register the Stylesheets for the admin area.
    *
-   * @since 2.0.0
+   * @since 1.0.0
    */
   public function enqueue_styles() {
 
-    $main_style = $this->general_helper->get_manifest_assets_data( 'applicationAdmin.css' );
-    wp_register_style( static::THEME_NAME . '-style', $main_style );
+    $main_style = General_Helper::get_manifest_assets_data( 'applicationAdmin.css' );
+    wp_register_style( static::THEME_NAME . '-style', $main_style, array(), static::THEME_VERSION );
```

Comment:
Wouldn't `static::THEME_VERSION` affect cache busting we are doing using `manifest.json`?

I know that there is an error that happens because theme version isn't specified, but there is an [upstream issue](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1472) opened at WPCS about this (they neglected the fact that hashing can be used).

You can either disable these lines with an inline comment 

`// phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion`

Or by adding this in the `phpcs.xml.dist` ruleset and disabling it.

Reply:
I have tested this and we are ok, assets manifest changes the name of the output file every time something changes so, everytime we hit build name of the file will change and cache busting will be done.

Reply:
Awesome, I thought that specifying version would suppress it. :+1: